services/user-management-services.yaml

```
@@ -1158,6 +1158,37 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /v1/clientApps/verifyApproval:
```

I would prefer it to be top-level endpoint, e.g.

```suggestion
  /v1/verifyClientApp:
```